validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java

```diff
@@ -173,6 +176,30 @@ public void checkIfIncomingBlockObeysSlashingConditions(BeaconBlock block) {
   }
 
   */
+
+  @Subscribe
+  public void onStoreInitializedEvent(final StoreInitializedEvent event) {
+    final Store store = chainStorageClient.getStore();
+    final Bytes32 head = chainStorageClient.getBestBlockRoot();
+    final BeaconState genesisState = store.getBlockState(head);
+
+    // Any deposits pre-genesis can be ignored.
+    newDeposits.clear();
```

Comment:
We probably need to keep block height info for deposits so we can just clear the events prior to genesis.  Seems like there's a race condition here.

Reply:
Agreed.  We don't handle ETH1 deposits right in a few ways at the moment so I've logged BC-159 to track this.  BC-158 to respect follow distance and BC-154 to actually calculate deposit proofs for post-genesis deposits.